src/function.ts

```diff
@@ -788,6 +795,24 @@ export class CallbackLambdaCode extends aws_lambda.Code {
                 }
               }
 
+              return o;
+            };
+
+                        /**
```

Comment:
Weird formatting

Reply:
hmmm, can't wait for prettier to support 4.7!